src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
It's generally not recommended to apply styles directly to the universal selector `*`. It can lead to unexpected results and performance issues, as it selects all elements in the DOM. It's better to use a CSS reset file or normalize.css to reset the default styles.